db/clean-python.bash

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Author: "abpwrs"
+# Date: 20200409
+
+# script:
+isort -rc .
+black .
```

Comment:
Could we incorporate black into our pipeline? Can we make a ticket for DB code linting for now.

What's isort?

Reply:
Yes, this could be done as a pre-commit, or pre-recieve hook. Added issue #96 

`isort` is a python import sorter.